test/fixtures_message_sender.js

```diff
@@ -565,8 +565,13 @@ module.exports = function() {
                     "to_identity": "cb245673-aa41-4302-ac47-00000001002",
                     "to_addr": "+27820001002",
                     "content": "Please reply to this message with your compliment. If it relates to the service at the clinic, include the clinic or clinic worker name. Standard rates apply.",
-                    "metadata": {}
-                }
+                    "metadata":{
+                      "template":{
+                        "name":"important_info",
+                        "variables":"Please reply to this message with your compliment. If it relates to the service at the clinic, include the clinic or clinic worker name. Standard rates apply."
```

Comment:
I would also expect a `language` field to be inside the `template`, this is probably because of the issue of where we're getting the language from that I mentioned.